src/Pages/MembershipApplication/registerPage.css

```diff
@@ -0,0 +1,277 @@
+@import url('https://fonts.googleapis.com/css2?family=Mina&display=swap%27');
+
+body {
```

Comment:
in react, adding a rule for `body` in one css file will apply it to the whole project, same as the rules below for h1, h2, etc.

to fix this lets not use generic selectors and instead use class names, a good example of this is [src/Pages/3DPrinting/3D-print-form.css:27](https://github.com/SCE-Development/Core-v4/blob/053a0dae3380f93319107e9ebd43e61d1fab07b2/src/Pages/3DPrinting/3D-print-form.css#L27)